R/pkg/R/functions.R

```diff
@@ -2156,28 +2178,23 @@ setMethod("date_format", signature(y = "Column", x = "character"),
             column(jc)
           })
 
-#' from_json
-#'
-#' Parses a column containing a JSON string into a Column of \code{structType} with the specified
-#' \code{schema} or array of \code{structType} if \code{as.json.array} is set to \code{TRUE}.
-#' If the string is unparseable, the Column will contains the value NA.
+#' @details
+#' \code{from_json}: Parses a column containing a JSON string into a Column of \code{structType}
+#' with the specified \code{schema} or array of \code{structType} if \code{as.json.array} is set
+#' to \code{TRUE}. If the string is unparseable, the Column will contains the value NA.
```

Comment:
btw, `will contains the value NA.` is very consistently documented. in this case this is right, but there are many other that says the value is `null` (note lower case) which isn't quite correct on the R side.

another project? :)